subworkflows/local/utils_nfcore_skierfe_pipeline/main.nf

```diff
@@ -28,68 +112,68 @@ include { workflowCitation          } from '../../nf-core/utils_nfcore_pipeline'
 
 workflow PIPELINE_INITIALISATION {
 
-    take:
-    version           // boolean: Display version and exit
-    help              // boolean: Display help text
-    validate_params   // boolean: Boolean whether to validate parameters against the schema at runtime
-    monochrome_logs   // boolean: Do not use coloured log outputs
-    nextflow_cli_args //   array: List of positional nextflow CLI args
-    outdir            //  string: The output directory where the results will be saved
-    input             //  string: Path to input samplesheet
+take:
```

Comment:
Looks like the indentation has disappeared for this code block

Reply:
Fixed.